scormxblock/constants.py

```diff
@@ -0,0 +1 @@
+SCORM_COMPLETE_STATUSES = ['completed', 'passed']
```

Comment:
It's okay to keep it singular i.e; `SCORM_COMPLETE_STATUS` or `SCORM_COMPLETION_STATUS`

Reply:
Done